src/orga/governance.rst

```diff
@@ -102,6 +108,68 @@ To call for a standard vote, here is a template PR comment:
     To vote please leave Approve (yea) or Request Changes (nay) reviews.
     If you would like changes to the current language please leave a comment or push to this branch.
     This vote will end on {date}.
+    
+----
+:Posting results: To maintain the historical record, the outcome of any standard vote should be 
+         recorded in the "vote-results" folder at 
+         https://github.com/conda-forge/conda-forge.github.io/tree/master/src/vote-results
+         
+         Each vote should be its own file.  The filename should reflect the topic and the 
+         date that the vote opened.  The file should contain at least:
+         
+         * vote description
+         * vote policy
+         * vote totals
+         * Poll open and close dates
+         
+         If a vote has "timed out," as described in "Quorum," the documentation regarding
+         notification records should also be in this file.
+         
+
+----
+:Quorum: All percentages below express *both* required participation, as a
+         fraction of the active core team, as well as the fraction of that
+         fraction who vote affirmatively on the issue. For example, in a vote
+         requiring 50%, with 18 active core members, at least 9 must vote;
+         if 9 vote, there must 5 affirmative votes. If 13 members vote, 7
+         must be affirmative.
+
+         Because everyone is busy and it is difficult sometimes to get quorum,
+         votes not achieving quorum will eventually time out. When this happens,
```

Comment:
```suggestion
         votes not achieving quorum will eventually time out on their set end date. When this happens,
```
Would this suffice to address https://github.com/conda-forge/conda-forge.github.io/pull/970#issuecomment-588362001 ?

Reply:
I think this is a good clarification.  It's less that timing out is a separate process, and more that it is an inevitable end, but in order to reach that end, the notification requirements must be met.

Reply:
Thanks!
@isuruf, does that address some of your concerns?

Reply:
Not really. We should then mention that secret/sensitive votes should have a minimum of 2 weeks.

Reply:
Hmm, we can't do that because of, e.g., https://github.com/conda-forge/conda-forge.github.io/pull/970/files#diff-aeeef496313214bb0e1ff8c044612ad3L175 when we would (unfortunately) have to have short sensitive votes.

So, I guess you mean https://github.com/conda-forge/conda-forge.github.io/pull/970#discussion_r380547258 is not clear enough -- do you have any alternative phrasing suggestions?

Reply:
If somebody had a 1 week vote and didn't get a quorum, it'll be in limbo, because after 1 week you can't vote, so the conditions for timing out cannot be fulfilled.

Reply:
Correct. That's what my "Additional requirements may apply \[...\]" hint at: If you want to make use of the new timeout rule, you have to meet the additional requirements (2+ weeks etc.). Shorter votes will be handled as is currently the case (i.e., no timeout can be applied).

Reply:
@isuruf, are you fine with the current state of the proposal then?
We are currently sitting at `14` out of `15` required approvals.